go.mod

```diff
@@ -210,3 +210,5 @@ require (
 	nhooyr.io/websocket v1.8.7 // indirect
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
+
+replace github.com/onflow/flow-go v0.33.2-0.20240425031927-4246f1af6031 => github.com/The-K-R-O-K/flow-go v0.33.2-0.20240507090417-9acec65fa54d
```

Comment:
If try to update this or remove, the bunch of errors come up

Reply:
this is OK in the first merge in `flow-emulator`. we'll submit a follow up PR to switch it to a commit in `flow-go` after your PR is merged